src/ts/Kryptoeuro.ts

```diff
@@ -24,12 +26,24 @@ class Kryptoeuro {
                 mobid.register(address, phoneNumber, ()=> {
                     localStorage.setItem(Kryptoeuro.LOCALSTORAGE_KEY_KEY, encryptedData);
                     this.eth.watchBalance(address);
-                    this.eth.watchEurBalance(address);
                     this.eth.watchBalanceGateway(address);
                 });
 
 
             }
+		this.ethAddress = address;
+                    this.eth.watchBalance(address);
+                    this.eth.watchBalanceGateway(address);
+		this.eth.readContractVarHexValue(address,"balanceOf(address)", function(res){
+			document.querySelector("#balance-data").innerHTML = String(parseInt(res,16)/100); 
```

Comment:
very old school way of working with dom, i think Framework7 have better options https://framework7.io/docs/template7-pages.html